lib/worker.js

```diff
@@ -106,7 +106,7 @@ Worker.prototype._run = function() {
         var modName = service.module || service.name;
         if (/^\./.test(modName)) {
             // resolve relative paths
-            modName = path.resolve(self._basePath + '/' + modName);
+            modName = path.resolve((self.config.config_dir || self._basePath), modName);
```

Comment:
So... In normal operation mode the config file location will be known. However, tests do some magic on config files, then require a `service-runner` and pass the config file directly as an object. Obviously, in this case the location of a config file is unknown, so we default back to our guessed `base_path`. 

This works, but I'm still not very happy about all the black magic happening here and open to suggestions. My be it would be better to completely prohibit passing the config as object to service-runner and in tests make config modifications and write the results into a temp file and pass it's path?